torchvision/transforms/_presets.py

```diff
@@ -16,6 +16,7 @@
     "VideoClassification",
```

Comment:
@TeodorPoncu Sorry I was wrong before about this file, I think this should be put inside prototype area, i.e. `torchvision/prototype/transforms/_presets.py`